plugin/pkg/auth/authorizer/rbac/rbac.go

```diff
@@ -62,7 +62,7 @@ func (r *RBACAuthorizer) Authorize(attr authorizer.Attributes) error {
 		}
 	}
 
-	return validation.ConfirmNoEscalation(ctx, r.authorizationRuleResolver, []rbac.PolicyRule{requestedRule})
+	return validation.ConfirmNoEscalation(ctx, r.authorizationRuleResolver, []rbac.PolicyRule{requestedRule}), nil
```

Comment:
for now, handle an error returned from ConfirmNoEscalation as a denial. @ericchiang, this is probably not how we want to do authorization checks for every request... escalation checks are a pretty heavyweight way to check specific attributes against a set of rules